src/notebook.h

```diff
@@ -434,6 +434,25 @@ class MKCAL_EXPORT Notebook
     */
     bool operator==(const Notebook &notebook) const;
 
+    /**
+      Retrieve the list of notebooks as stored in the system.
+
+      @param account when not null restrict the list to a given
```

Comment:
Code is using isEmpty(), not isNull(), and string emptiness is a cleaner concept. Could adjust the comment here?

Reply:
Sure, changed for empty wording.